### PR TITLE
Minutes of meeting 2024-06-19

### PR DESCRIPTION
**Pull request by sthagen**
- [x] initial pre-meeting draft
- [x] synced with roster
- [x] synced with agenda
- [x] post-meeting draft
- [x] ready for review

### CODE REVIEW

**Review by santosomar — APPROVED** [review]
Thank you for setting the agenda and skeleton of the meeting notes @sthagen !!

File: meeting-minutes/2024-06-19.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+## 3.2 Review pending Pull Requests
+
+* [Information Model example #24](https://github.com/oasis-tcs/openeox/pull/24)
+  * Omar presents the proposal (David Kemp is not present due to public holiday in US)
+  * Thomas Schmidt sees much more complexity and not much improvement
+  * Przemyslaw seconds and adds tha deriving YAML from the JSON is trivial
+  * Stefan Hagen adds as previously discussed:
+    * the target OpenEOX data model will be nearly flat with simple types and not nested with difficult types
+      so no big value assumed but high cost in readability
+    * JSON schema 202-12 is matching OpenAPI 3.1 so implementers already get a good start and JSON is easy to transcribe
```
Comment (tschmidtb51) [inline meeting-minutes/2024-06-19.md:117]:
```suggestion
    * JSON schema 2020-12 is matching OpenAPI 3.1 so implementers already get a good start and JSON is easy to transcribe

File: meeting-minutes/2024-06-19.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  * Expectation management is that we will not find terms that will be perfect for all vendors
+    as many vendors already use terms in an uncoordinated and sales oriented way
+  * Pablo articulates that an important outcome should be that the customer may still have to deal with 5 different
+    support tiers but he can map these to a simple end of life model without being forced to learn the life cycle
+    of every vendor first.
+  * Przemyslaw asks if there are use cases where there are more than one tier applicable at one time for one product
+  * Stefan mentions Ubuntu OS distribution versus security patches for select components only offered for a pro tier
+  * All discuss what to map to end of support as an example as to how to apply a unified schema onto complete endpoint
+    system environment
+  * Sonny mentions a delivery with soft - and hardware components evolving along different life cycles - maybe
```
Comment (tschmidtb51) [inline meeting-minutes/2024-06-19.md:137]:
```suggestion
  * Sonny mentions a delivery with soft- and hardware components evolving along different life cycles - maybe

File: meeting-minutes/2024-06-19.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    not able to see this always as a single endpoint
+  * Omar demonstrates as example a system hardware that contains Red Hat Enterprise Linux and an OpenSSL version not
+    supported by Red Hat inside and shows how he tried to embody this in the PR #13
+  * Omar will continue to work on the PR #13
+* [BSI scenarios #26](https://github.com/oasis-tcs/openeox/pull/26)
+  * Thomas Schmidt shortly presents the use case templates from BSI to receive feedback from the TC
+    with the goal that anyone else can use these tempates to describe their use cases / scenarios
```
Comment (tschmidtb51) [inline meeting-minutes/2024-06-19.md:144]:
```suggestion
    with the goal that everyone else can use these templates to describe their use cases / scenarios

File: meeting-minutes/2024-06-19.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    in an easy to compare and combine way
+  * All agree to review the contributions from the outside in and only provide formal suggestions that optimize comparability
+    (not discuss the content of scenarios of other suppliers)
+  * Omar suggests that he and his team take a look and we follow this shallow verification approach to collect
+    the needed data fast and then take time to discuss the combination of the provided use cases / scenarios
+    * No objections, al in favor
```
Comment (tschmidtb51) [inline meeting-minutes/2024-06-19.md:150]:
```suggestion
    * No objections, all in favor

**Review by tschmidtb51 — COMMENTED** [review]
Thank you for providing the minutes right after the meeting!

Just a few editorial nits

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Review by justmurphy — APPROVED** [review]
LGTM

**Pull request merged**